concordium-sdk/src/main/java/com/concordium/sdk/crypto/wallet/web3Id/Statement/RequestStatement.java

```diff
@@ -13,4 +16,22 @@ public class RequestStatement {
     private String id;
     private List<String> type;
     private List<AtomicStatement> statement;
+
+    public List<AtomicStatement> getUnsatisfiedStatements(IdentityObject identityObject) {
```

Comment:
Should this check in some way that the statement is actually about an account credential?